services/api/src/authentication/guards/github.guard.ts

```diff
@@ -0,0 +1,18 @@
+import { Injectable, Logger } from "@nestjs/common";
+import { AuthGuard } from "@nestjs/passport";
+
+@Injectable()
+export class GithubAuthGuard extends AuthGuard("github") {
+  constructor(private readonly logger: Logger) {
+    super();
+  }
+
+  handleRequest(err: unknown, user: unknown, info: { message?: string }): any {
+    if (err || !user) {
+      this.logger.verbose(err || info.message);
```

Comment:
Do we want to keep the log here?
I guess verbose logging while never be used in prod. If we want to keep it I'd add the same behaviours to the twitter guard.